src/components/Feedback/Feedback.tsx

```diff
@@ -38,7 +44,7 @@ const Feedback: React.FC<any> = ({ location }) => {
     values: React.ComponentProps<typeof FeedbackBox>["onSubmit"]
   ) => {
     submitFeedback({
-      feedback: values.comment,
+      feedback: isError ? `${statusCode} - ${values.comment}` : values.comment,
```

Comment:
Looks like statusCode could be null so we may just want to have a default. What are your thoughts? 

`{statusCode ?? 'Unknown'} - ${values.comment}`

Reply:
We may also want to prefix the status code with something like Error Code: or HTTP Status Code: just so folks understand what the status code means!

Reply:
Yes, good idea!